azurelinuxagent/ga/exthandlers.py

```diff
@@ -223,7 +223,15 @@ def run_status(self):
         return
 
     def get_guid(self, name):
-        return self.last_guids.get(name, None)
+        return self.last_guids.get(name, (None, False))[0]
+
+    def get_log_guid(self, ext_handler):
```

Comment:
consider renaming `log_guid`, I think it is a bit misleading

Reply:
Fixed.